packages/components/molecules/f-restaurant-card/CHANGELOG.md

```diff
@@ -3,7 +3,14 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-v1.6.1
+## v1.6.2
+
+_April 10, 2024_
+
+### Changed
+- Create prop for max rating value
```

Comment:
please can provide a bit more context for future visitors to the repo about what this change does?